src/modules/reserve/getReserverations.js

```diff
@@ -0,0 +1,8 @@
+const getReserverations = async (movieId) => {
+  const request = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/sUHYcv06blVrKgUc2kUH/reservations?item_id=${movieId}`);
+  const response = await request.json();
+
+  return response;
+};
```

Comment:
#### Great job on your reservation popup team. But I suggest you address the following. 
- [ ] When a user clicks on the "Reserve" button, beside the data being recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), the screen should be updated as a result with the new reservation we make.
- [ ] Your Reservation seems to be falling for some of the movies. Please make sure it's possible to reserve any of the movies that we have listed.
- [ ] Also, As the wire frame suggested the pop-up better be centered on the page.
- [ ] I suggest adding a reservation counter on the pop-up window, and the value should be incremented accordingly. when we add a new reservation the reservation count should be incremented and the screen is populated with the new count. Remember, The counter should be implemented as a separate module.
- [ ] Once we have the reservation counter, we need to cover the counter function with unit tests using Jest. Make sure that you have tests for all edge cases.

https://user-images.githubusercontent.com/69077061/221371435-3011de83-9da5-43da-a267-10d7f7f50214.mp4